core/lerna/package.json

```diff
@@ -44,6 +44,7 @@
     "@lerna/diff": "file:../../commands/diff",
     "@lerna/exec": "file:../../commands/exec",
     "@lerna/import": "file:../../commands/import",
+    "@lerna/info": "file:../../commands/info",
```

Comment:
You'll need to run `npm i` in the root of the repo, as well. There should be a new entry in `package-lock.json`